src/tests/ftest/process_core_files.py

```diff
@@ -377,28 +366,33 @@ def delete_gdb_core_files(self):
             int: number of errors
 
         """
-        self.log.debug("Checking core files generated by core file processing")
+        log = logging.getLogger()
```

Comment:
The more I see this, the more I wonder if there is a proper way to set globally in files. But I'm not familiar enough to know